src/sentry/api/event_search.py

```diff
@@ -946,6 +1016,10 @@ def resolve_field(field):
     if not isinstance(field, six.string_types):
         raise InvalidSearchQuery("Field names must be strings")
 
+    match = is_function(field)
+    if match:
+        return resolve_function(field, match)
```

Comment:
It would be nice to consolidate this path and the the sans_parens path in the future.

Reply:
Yeah, I think there can be follow up PRs to completely strip all the magic function field aliases out of the code, along with these checks.